src/new/sendTransaction.js

```diff
@@ -6,18 +6,15 @@
  */
 
 import EventEmitter from 'events';
-import { CREATE_TOKEN_TX_VERSION, MIN_POLLING_INTERVAL, SELECT_OUTPUTS_TIMEOUT } from '../constants';
+import { MIN_POLLING_INTERVAL, SELECT_OUTPUTS_TIMEOUT, HATHOR_TOKEN_CONFIG } from '../constants';
 import transaction from '../transaction';
+import helpers from '../utils/helpers';
 import txApi from '../api/txApi';
 import txMiningApi from '../api/txMining';
-import { AddressError, OutputValueError, ConstantNotSet, MaximumNumberOutputsError, MaximumNumberInputsError } from '../errors';
+import { WalletError, SendTxError, OutputValueError, ConstantNotSet, MaximumNumberOutputsError, MaximumNumberInputsError } from '../errors';
 import wallet from '../wallet';
+import oldHelpers from '../helpers';
 import storage from '../storage';
-import Input from '../models/input';
-import Output from '../models/output';
-import Transaction from '../models/transaction';
-import CreateTokenTransaction from '../models/create_token_transaction';
-import Address from '../models/address';
 import MineTransaction from '../wallet/mineTransaction';
 
 /**
```

Comment:
Comment not related to this line.

Should we create an interface that is implemented by both `SendTransaction` classes?

Reply:
When I started doing this task I created the interface but javascript code does not have any interface implementation. Because of that, we would not be able to use it to enforce the same method returns/signatures on it, then I decided not to do it. 

I can create it and the classes on typescript may implement it but the ones still using javascript wouldn't benefit from it.

(This comment is valid for both `SendTransaction` and `HathorWallet` classes)

Reply:
I got it. I still think it is valuable to have an explicit interface implemented by all classes. I have the same considerations that I told in the other conversation.

Reply:
Makes sense, done here https://github.com/HathorNetwork/hathor-wallet-lib/pull/221/commits/9dd1b760fdfdce10f2f4022fad206998e1777527

Reply:
Well done! Thanks!